content/en/docs/tasks/tools/_index.md

```diff
@@ -24,8 +24,8 @@ Find your preferred operating system below.
 ## kind
 
 [`kind`](https://kind.sigs.k8s.io/) lets you run Kubernetes on
-your local computer. This tool requires that you have
-[Docker](https://docs.docker.com/get-docker/) installed and configured.
+your local computer. This tool requires that you have either
+[docker](https://www.docker.com/) or [podman](https://podman.io/) installed.
```

Comment:
Use title case for proper nouns.
```suggestion
[Docker](https://www.docker.com/) or [Podman](https://podman.io/) installed.
```

Reply:
Thanks @sftim. I mentioned you because you're the author of the issue. So I guess it wasn't necessary? 
Sorry, I'm sure what's the protocol/etiquette when you work on somebody's issue and you want to get their feedback.